.chloggen/xrayexporter-dot-converter-featuregate-beta.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awsxrayexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: change x-ray exporter's translator to make "." split annotation pass as-is
```

Comment:
This should mention the feature gate name, and the fact that it's being upgraded from `Alpha` to `Beta`, along with the description you already have here.

Reply:
Done, thanks